docs/sources/open-source/_index.md

```diff
@@ -90,10 +90,6 @@ features:
       callback_id: add_resolution_note
       description: Add this message to resolution note
   slash_commands:
-    - command: /oncall
```

Comment:
I guess a TODO for after to update the existing app manifests in slack

Reply:
thanks for the reminder 👍